client/src/components/Burger/Burger.js

```diff
@@ -0,0 +1,55 @@
+import React, { Component } from "react";
+import "./Burger.css";
+
+// Class Based React Component
+class Burger extends Component {
+  constructor(props) {
+    super(props);
+    console.log(props);
+
+    // Default CSS class to apply to the Component
+    this.state = {
+      classList: "Burger",
+      active: false
+    };
+  }
+
+  clicked = () => {
+    this.setState({ active: !this.state.active })
+    // Utilizes a callback in the Navbar to display the menu.
+    // this.props.DisplayMenu()
```

Comment:
Can you set this up to take and invoke the a callback?

Reply:
This should now be set up as we discussed.